system/lib/libc/lookup_name.c

```diff
@@ -0,0 +1,27 @@
+// Emscripten-specific version of musl/src/network/lookup_name.c
+
+#include <string.h>
+#include "musl/src/network/lookup.h"
+#include "emscripten_internal.h"
+
+int __lookup_name(struct address buf[static MAXADDRS], char canon[static 256], const char *name, int family, int flags)
+{
+	if (family != AF_INET || flags != AI_CANONNAME) {
+		return EAI_SYSTEM;
+	}
+
+	*canon = 0;
+	if (name) {
+		/* reject empty name and check len so it fits into temp bufs */
+		size_t l = strnlen(name, 255);
+		if (l-1 >= 254)
```

Comment:
```suggestion
		if (l >= 255)
```
Or am I missing something?

Reply:
This code is copied directly from the musl's lookup_name.c: https://github.com/emscripten-core/emscripten/blob/d01d57e0b432ab209021bd47380d87bb1f8b647c/system/lib/libc/musl/src/network/lookup_name.c#L312-L319
 
I will add a comment to that effect.